tests/icelake_config_id/host/host.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+#include "icelake_config_id_u.h"
+
+int main(int argc, const char* argv[])
+{
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+
+    if (argc < 2)
+    {
+        fprintf(
+            stderr,
+            "Usage: %s ENCLAVE_PATH [--host-threads n] [--enclave-threads n] "
+            "[--ecalls]\n",
+            argv[0]);
+        return 1;
+    }
+
+    const uint32_t flags = oe_get_create_flags();
+
+    oe_sgx_enclave_setting_config_data config_data_setting_optional = {
+        {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15},
+        2,
+        true};
```

Comment:
Consider adding comment to explain the Boolean value.
```suggestion
        true /* ignore_if_unsupported */};
```